src/common/translations/localize.ts

```diff
@@ -116,13 +94,21 @@ export const computeLocalize = async <Keys extends string = LocalizeKeys>(
       } catch (err: any) {
         return "Translation error: " + err.message;
       }
-      cache._localizationCache[messageKey] = translatedMessage;
+      localizationCache[translatedValue] = translatedMessage;
     }
 
     let argObject = {};
+    // TS says that we will always get the new format, but we might get the old format
+    const args = _args as any;
     if (args.length === 1 && typeof args[0] === "object") {
       argObject = args[0];
-    } else {
+      if (Object.values(argObject).some((v) => v === undefined)) {
+        // eslint-disable-next-line no-console
+        console.warn("[FIXME] While localizing", key, "undefined was passed");
+      }
```

Comment:
Why is this an issue?

Reply:
Because
1. According to TS it should be impossible to pass undefined
2. If undefined is passed, with the next localize library it'll show up as `{value}` instead of empty